src/utils/string/index.ts

```diff
@@ -228,3 +228,6 @@ export const joinWithConjunction = (array: string[], conjunction: string) => {
  * @return {string} string with new lines removed
  */
 export const stripNewLines = (str: string) => str.replace(/\n/g, "");
+
+export const toSentenceCase = (string: string) =>
+  string[0].toUpperCase() + string.substring(1).toLowerCase();
```

Comment:
Would you mind adding a test for this function in the corresponding test file? I promise it'll only take 2 seconds 👀

Reply:
true, it only took two seconds. good call!